crates/cxx-qt-gen/src/parser/constructor.rs

```diff
@@ -0,0 +1,126 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+use syn::{
+    Path,
+    AngleBracketedGenericArguments, Error, ItemImpl,
+    PathArguments, PathSegment, Result,Type, GenericArgument,
+    spanned::Spanned, ImplItem
+};
+
+#[derive(Default)]
+pub struct ImplItems {
```

Comment:
some doc lines about the fields could be useful here

Reply:
Done.